Vector/Model/Room/RoomDataSource.m

```diff
@@ -42,4 +47,53 @@ - (instancetype)initWithRoomId:(NSString *)roomId andMatrixSession:(MXSession *)
     return self;
 }
 
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    UITableViewCell *cell = [super tableView:tableView cellForRowAtIndexPath:indexPath];
+    
+    // Finalize cell view customization here
+    if ([cell isKindOfClass:MXKRoomBubbleTableViewCell.class])
+    {
+        MXKRoomBubbleTableViewCell *bubbleCell = (MXKRoomBubbleTableViewCell*)cell;
+        
+        // Display timestamp for the last message.
+        if (indexPath.row == [tableView numberOfRowsInSection:0] - 1)
+        {
+            if (bubbleCell.bubbleData.bubbleComponents.count)
+            {
+                [bubbleCell addTimestampLabelForComponent:bubbleCell.bubbleData.bubbleComponents.count - 1];
+            }
```

Comment:
how can we be sure that the timestamp must be rendered on the last item ?
Is it possible to have an index in bubble cell for the dedicated container.